integration-test/src/main/java/org/apache/iotdb/it/env/cluster/env/AbstractEnv.java

```diff
@@ -242,7 +243,30 @@ public String getTestClassName() {
     return "UNKNOWN-IT";
   }
 
-  public void testWorking() {
+  private Map<String, Integer> countNodeStatus(Map<Integer, String> nodeStatus) {
+    Map<String, Integer> result = new HashMap<>();
+    nodeStatus.values().forEach(status -> result.put(status, result.getOrDefault(status, 0) + 1));
+    return result;
+  }
+
+  private final Predicate<Map<Integer, String>> statusCheckNoUnknown =
+      nodeStatus -> countNodeStatus(nodeStatus).getOrDefault("Unknown", 0) == 0;
+
+  private final Predicate<Map<Integer, String>> statusCheckOneUnknownThreeRunning =
+      nodeStatus -> {
+        Map<String, Integer> count = countNodeStatus(nodeStatus);
+        return count.getOrDefault("Unknown", 0) == 1 && count.getOrDefault("Running", 0) == 3;
+      };
+
+  public void testWorkingAllFine() {
+    testWorking(statusCheckNoUnknown);
+  }
+
+  public void testWorkingSpecial() {
+    testWorking(statusCheckOneUnknownThreeRunning);
+  }
+
+  private void testWorking(Predicate<Map<Integer, String>> statusCheck) {
```

Comment:
I think the `testWorking` can be made to public now. Therefore we can use the input `Predicate` to simplify other integration-tests in the future~

Reply:
Constructive feedback !